runtime/moonbase/src/runtime_params.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 Moonbeam Foundation.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Dynamic runtime parametes.
+use crate::Runtime;
+use frame_support::dynamic_params::{dynamic_pallet_params, dynamic_params};
+use sp_runtime::Perbill;
+
+#[dynamic_params(RuntimeParameters, pallet_parameters::Parameters::<Runtime>)]
+pub mod dynamic_params {
+	use super::*;
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod runtime_config {
+		// for fees, 80% are burned, 20% to the treasury
+		#[codec(index = 0)]
+		pub static FeesTreasuryPercentage: Perbill = Perbill::from_percent(20);
```

Comment:
This could probably be `Percent` instead of `Perbill`. But not an issue.

Reply:
It's to give ability to add more precision if needed in the future.